<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingWindow</string>
  </property>
  <widget class="QLabel" name="Label_title">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_savesetting">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting Save</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="TabSetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Tab_basic">
    <attribute name="title">
     <string>Basic set</string>
    </attribute>
    <widget class="QLabel" name="label_text1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max dist ( mm) :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_maxdist">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_text3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Angle Choice :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_90">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>98</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>90 (Solo)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_180">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>180 ( Multi )</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_text2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2000mm (recommend)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_text4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Buzz Sound Duration :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_buzzduration">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_text5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>second</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_text4_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequeny of getting Lidars datas :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_lidar_fps">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_text6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FPS</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Tab_port">
    <attribute name="title">
     <string>port</string>
    </attribute>
    <widget class="QLabel" name="Label_port_top">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>60</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Top :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_port_mid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>60</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Mid :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_port_bot">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>60</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Bot :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_port_top">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_port_mid">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_port_bot">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Label_explain">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>메인 메뉴의 Data View에서
인식되는 라이다의 상,중,하 위치를 보고
필요 시 Port 위치를 재배열하십시오.
또한 아래의 포트 리스트를 보고 1,2,3을 재배열 하십시오,</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="FuncLable_portlist">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>error</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_portlist">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>60</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port List :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Tab_env">
    <attribute name="title">
     <string>Environment</string>
    </attribute>
    <widget class="QListWidget" name="List_env">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>186</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="FuncLabel_selected_env">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>selected model file</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Tab_model">
    <attribute name="title">
     <string>AI Model</string>
    </attribute>
    <widget class="QLabel" name="FuncLabel_selected_model">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>selected model file</string>
     </property>
    </widget>
    <widget class="QListWidget" name="List_model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>186</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
